test/Spec.hs

```diff
@@ -25,7 +26,12 @@ main :: IO ()
 main = do
     hSetEncoding stdout utf8
     hSetEncoding stderr utf8
-    dbPool <- Pool.createPool (Sql.connectPostgreSQL connectionSettings) Sql.close 10 5 10
+    dbPool <- 
+#if MIN_VERSION_resource_pool(0,4,0)
+        Pool.newPool (Pool.defaultPoolConfig ((Sql.connectPostgreSQL connectionSettings)) Sql.close 1 10)
```

Comment:
Are you sure? Wasn't `newPool` added in v0.3.0.0?

Reply:
It was, but at that point there was no defaultPoolConfig :)

So I'm doing minimum amount of changes to make it work warning-less with the versions we allow in cabal file.

Reply:
Ah I see. 👍